core/bootstrap.c

```diff
@@ -0,0 +1,215 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2013, 2014 Intel Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
```

Comment:
This code does not originate from Intel so the Copyright header must be changed.